yoshihara/yyyymmddesa
# Reimplementation

- Use newest node

## Implement

- Re implement about cache
    - Before: each prev/this/next month's posts are cached and restored
    - After: `this-2` ~ `this + 2` month's posts are cached at once
- Simplify (remove needless classes/import)

## Test

- Setup jest
- Add tests
- Add CircleCI configuration
- Check outdated & newest even major version packages every monday

## Developement

- Setup babel
- Use npm instead of yarn
- Use eslint & prettier
- Update packages (including webpack 3 -> 4)
